endpoints/apiconfig.go

```diff
@@ -777,7 +796,16 @@ func validateField(v reflect.Value, t reflect.StructField) error {
 
 	isZero := v.Interface() == reflect.Zero(v.Type()).Interface()
 	if isZero && tag.required {
-		return fmt.Errorf("missing field %v", t.Name)
+		return fmt.Errorf("missing field %v in %v", t.Name, parent.Name())
+	}
+
+	if v.Kind() == reflect.Struct {
+		if v.CanAddr() && hasExportedFields(v) {
```

Comment:
Have you seen any case where the value is not addressable? In those cases, should we return an error instead of just ignoring it silently?